mod/likes/views/default/likes/css.php

```diff
@@ -7,9 +7,8 @@
 /* ***************************************
 	Likes
 *************************************** */
-.elgg-likes {
+.elgg-likes-popup {
```

Comment:
elgg-likes is no longer used and replaced by popup. If we would reuse elgg-likes we would have conflicting css (as it declared position absolute; which causes issues in the lightbox)